Library/Basic Set/Basic Set Advantages.adq

```diff
@@ -18206,4 +18206,137 @@
 			<category>Advantage</category>
 		</categories>
 	</advantage>
+	<advantage version="3">
+		<name>Alternate Form</name>
+		<type>Physical, Exotic</type>
+		<base_points>15</base_points>
+		<modifier version="1" enabled="no">
+			<name>Cosmetic</name>
+			<cost type="percentage">-50</cost>
+			<affects>total</affects>
+			<reference>B84</reference>
+		</modifier>
+		<modifier version="1" enabled="no">
+			<name>Absorptive Change</name>
+			<cost type="percentage">5</cost>
+			<levels>1</levels>
+			<affects>total</affects>
+			<reference>P75</reference>
+			<notes>@Level of Absorbptive Change. 1 None, 2, Light, 3, Medium, 4, Heavy, 5, Extra Heavy@</notes>
```

Comment:
I believe "Absorbptive" is a misspelling.

Reply:
Fixed, thanks